src/components/form-elements/_select.scss

```diff
@@ -1,40 +1,43 @@
-$selectHeight: rhythm(3 / 2);
-$selectBackground: $white;
+$selectHeight: 40px;
+$selectBackground: $graySecondaryLightest;
 $selectTextColor: $grayPrimary;
 $selectFontSize: 16px; // fix zooming to select on focus in iOS devices
-$selectIconSize: rhythm(1 / 3);
-$selectBorderRadius: 7px;
-$selectActiveBorderColor: $grayPrimary;
+$selectBorderRadius: 20px;
 $selectActiveInvalidBorderColor: $peachPrimaryDark;
 $selectActiveValidBorderColor: $mintPrimaryDark;
-$arrowColor: $graySecondary;
+$selectLargeBorderRadius: 28px;
+$selectLargeFontSize: 18px;
+$selectLargeHeight: 56px;
 
 $includeHtml: false !default;
 
 @if ($includeHtml) {
   .sg-select {
-    @include component();
+    display: inline-block;
+    position: relative;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    margin: 0;
     font-size: $selectFontSize;
     color: $selectTextColor;
-    box-shadow: $componentShadow;
     border-radius: $selectBorderRadius;
     height: $selectHeight;
+    border: 2px solid $graySecondaryLightest;
 
     &__element {
+      position: relative;
+      top: -2px;
```

Comment:
how about extract that to some `$borderRelatedVariable`? that would be clear what's happening here without need to check the border

Reply:
👍